### ISSUE TITLE
Create new sections with grouped table

### ISSUE THREAD

**Issue by NeviDimka** [bvogelzang/BVReorderTableView#2]
can you make reorder to work on grouped tableview please.
Thanks in advance!

P.S. Really not works only when drag to empty section

**Comment by NeviDimka** [comment]
yes.. as I wrote little later - it works for grouped table, thanks!
but can't get it to move an cell to empty section. Need to have at least one cell in group to move any other item to it

**Comment by bvogelzang** [comment]
Thanks for the clarification. I'll try to get new section creation working at some point soon.

**Comment by NeviDimka** [comment]
Hi! any update on this issue? I like your solution.. its simple and works fine... and if this issue will be fixed - it will be best solution for me :)) THANKS
